### DISCUSSION TITLE
Telegram topics support

### DISCUSSION THREAD

**Discussion by hoboroten** [grafana/grafana#68527] [Alerting]
Hi.
Recently Telegram added support for Topics in Groups in Bot API. Will it be added to Telegram Alert Channel configuration? As far as i can see - there must be an opportunity to set message_thread_id parameter.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82692. Feel free to continue the discussion around this there. Thank you!
